ansible_collections/arista/avd/molecule/eos_designs_unit_tests/documentation/devices/DC1-BL1A.md

```diff
@@ -1,1023 +0,0 @@
-# DC1-BL1A
```

Comment:
is it expected that all the documentation for devices is gone?

Reply:
fixed